keras/backend/theano_backend.py

```diff
@@ -782,7 +782,7 @@ def dropout(x, level, seed=None):
     if level < 0. or level >= 1:
         raise Exception('Dropout level must be in interval [0, 1[.')
     if seed is None:
-        seed = np.random.randint(10e6)
+        seed = np.random.randint(1,10e6+1) 
```

Comment:
1) you do realize that `randint(1, 10e6)` and `randint(1, 10e6 + 1)` are no different for this purpose, right?
2) this is not PEP8
3) is this an actual problem that you've run into?